pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -216,6 +215,12 @@ func (d *dpiComponent) dpiEnvVars() []corev1.EnvVar {
 		{Name: "DPI_TYPHACERTFILE", Value: d.cfg.TyphaNodeTLS.NodeSecret.VolumeMountCertificateFilePath()},
 		{Name: "DPI_TYPHAKEYFILE", Value: d.cfg.TyphaNodeTLS.NodeSecret.VolumeMountKeyFilePath()},
 		{Name: "DPI_FIPSMODEENABLED", Value: operatorv1.IsFIPSModeEnabledString(d.cfg.Installation.FIPSMode)},
+
+		{Name: "CLUSTER_NAME", Value: d.cfg.ESClusterConfig.ClusterName()},
+
+		{Name: "LINSEED_CLIENT_CERT", Value: d.cfg.DPICertSecret.VolumeMountCertificateFilePath()},
+		{Name: "LINSEED_CLIENT_KEY", Value: d.cfg.DPICertSecret.VolumeMountKeyFilePath()},
+		{Name: "FIPS_MODE_ENABLED", Value: operatorv1.IsFIPSModeEnabledString(d.cfg.Installation.FIPSMode)},
```

Comment:
Isn't this redundant with `DPI_FIPSMODEENBLED`?

Reply:
Yup, I kinda find it simpler to have FIPS_MODE_ENABLED everywhere. I will make the change in dpi.

Reply:
We should remove setting DPI_FIPSMODEENABLED in this PR then instead of having both env vars set